### PR TITLE
add XRButton type

### PR DESCRIPTION
**Pull request by Yonet**
### Why

Add missing XRButton type

### What

Add new type

### Checklist

-   [x] Checked the target branch (current goes `master`, next goes `dev`)
-   [x] Added myself to contributors table
-   [x] Ready to be merged

### CODE REVIEW

File: types/three/examples/jsm/webxr/XRButton.d.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { WebGLRenderer } from '../../../src/Three';
+
+export namespace XRButton {
+    function createButton(renderer: WebGLRenderer): HTMLElement;
+}
```
Comment (Methuselah96) [inline types/three/examples/jsm/webxr/XRButton.d.ts]: Can you make this a class with a static method?

**Review by Methuselah96 — CHANGES_REQUESTED** [review]
Thanks for the PR!

Can you add a simple test for this?

File: types/three/examples/jsm/webxr/XRButton.d.ts (hunk 1 above)
Comment (Yonet) [reply types/three/examples/jsm/webxr/XRButton.d.ts]: Sure!

Comment (Yonet) [reply types/three/examples/jsm/webxr/XRButton.d.ts]: @Methuselah96 I've changed it into a class and update the tests. When I run yarn test, I get the below error
'no-unnecessary-class  Every member of this class is static. Use namespaces or plain objects instead.'
Let me know how I should resolve.
Thanks!

Comment (Methuselah96) [reply types/three/examples/jsm/webxr/XRButton.d.ts]: Huh, interesting. That seems like a strange lint rule to have for declaration files and seems to have more to do with original code. I would disable with a comment right above the class declaration with `// tslint:disable-next-line:no-unnecessary-class`.

**Pull request closed without merging**